### PR TITLE
Add the F# Discord

### PR DESCRIPTION
**Pull request by Happypig375**

### CODE REVIEW

**Review by vzarytovskii — APPROVED** [review]

**Review by WhiteBlackGoose — APPROVED** [review]

File: _layouts/default.html [hunk 1]
```diff
@@ -142,6 +142,9 @@
                             <li>
                                 <a href="/guides/slack">F# on Slack</a>
                             </li>
+                            <li>
```
Comment (dsyme) [inline _layouts/default.html:145]: Please also add to index.html (these are duplicated)

**Review by dsyme — APPROVED** [review]

**Comment by voronoipotato** [comment]
Thanks Don! Can we make it more clear that its unofficial and not managed by the FSSF?

**Comment by dsyme** [comment]
Ah I see.  Who has admin rights?  Any chance we can bring this under FSSF?

"F# on Twitter" link is somewhat similar mind you.  

Any idea how we should distinguish?

**Comment by vzarytovskii** [comment]
> Ah I see. Who has admin rights? Any chance we can bring this under FSSF?
> 
> "F# on Twitter" and F# on StackOverflow" and "F# Code Review" links are somewhat similar mind you (though I understand there might be particular concerns about Discord)
> 
> Any idea how we should distinguish?

@dsyme 
Probably will make sense to distinguish as a "Community Server" or something along the lines?

**Comment by dsyme** [comment]
Maybe 

    "F# on Discord (non-FSSF)"
    "F# on StackOverflow (non-FSSF)"

etc.

**Comment by voronoipotato** [comment]
An (unofficial) or (not affiliated with the F# Foundation) might help. @Happypig375 what do you think? 

I am in the discord and have been for over a year now, it seems pretty on topic and healthy. A few board members are okay with it being referenced as unaffiliated contingent on it not violating CoC, but no unanimous opinion yet. Reed raised concerns about splitting the community, and while valid I also have a fear that pure centralization could limit growth. I believe the owner is woofdog and canine is one of the mods. I'm not sure that bringing it under the FSSF management is necessarily the best course of action but it would be good to have a relationship with the owners.

**Comment by dsyme** [comment]
I see thanks.

**Comment by dsyme** [comment]
Is the F# Speakers Program affiliated?  The F# User Groups map?  The ones on community.fsharp.org

**Comment by dsyme** [comment]
How about "fssnip.net"?

**Comment by voronoipotato** [comment]
The Speakers Program is affliated and the User Groups are also affiliated. I believe fssnip is not affiliated but we do have a relationship with the owner @tpetricek .

**Comment by dsyme** [comment]
OK thanks

**Comment by dsyme** [comment]
See https://github.com/fsharp/fsfoundation/pull/878

I also added one to the "Videos" link (which now points to a "learn F#" search on YouTube, since the old videos lists were way too out-of-date and impossible to keep fresh).

**Comment by dsyme** [comment]
> @dsyme Do you think, it also makes sense to add the FSSF official YT channel? We have a bunch of stuff there from compiler sessions and community review - https://www.youtube.com/c/fsharporg

Absolutely

**Comment by dsyme** [comment]
Done

**Pull request merged**